api/query_builder_test.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+import "testing"
+
+func TestPullRequestGraphQL(t *testing.T) {
+	tests := []struct {
+		name   string
+		fields []string
+		want   string
+	}{
+		{
+			name:   "empty",
```

Comment:
I think we could add one additional test for when the fields requested are invalid.